src/components/TypeComponents/SimpleSearch/Widget/simpleSearchPathBuilder.js

```diff
@@ -84,16 +84,20 @@ const simpleSearchPathBuilder = (widgetDef, widgetConf, stripes) => {
 
       // Then take each of the rules within the filter, and OR them together with the correct comparators
       const { rules } = f;
+
       rules.forEach((r, ind) => {
-        if (r.comparator === 'isNull' || r.comparator === 'isNotNull') {
+        if (r.comparator === 'isNull' 
+        || r.comparator === 'isNotNull' 
+        || r.comparator === 'isSet' 
+        || r.comparator === 'isNotSet') {
           // If we're allowing null the filterString is slightly different
           specificFilterString += `${filterPath} ${r.comparator}`;
+
         } else {
-          // Ensure we're safely encoding all special characters into the filters path
-          const encodedFilterValue = encodeURI(r.filterValue);
-          specificFilterString += `${filterPath}${r.comparator}${tokens(encodedFilterValue, stripes)}`;
+         // Ensure we're safely encoding all special characters into the filters path, after applying tokens
+         const encodedFilterValue = encodeURI(tokens(r.filterValue, stripes));
```

Comment:
Where did this change come from? 

This is ERM-1658, which I had on a branch somewhere, but it shouldn't have been available yet, is this something you've implemented yourself?

Reply:
I think I was trying to have the latest version of code, and by mistake I copied this part of code from ERM-1658

Reply:
I make sure to have the old encodedFilterValue for this line of code.